src/pages/en/guides/deploy/vercel.md

```diff
@@ -17,6 +17,17 @@ Your Astro project can be deployed to Vercel as a static site, or as a server-si
 
 Your Astro project is a static site by default. You don’t need any extra configuration to deploy a static Astro site to Vercel. 
 
+:::note
+There is currently a Vercel issue displaying a 404 page on Astro websites. Until this is fixed, you can add the following config file at the root of your project:
+
+```json title="vercel.json"
+{
+  "cleanUrls": true,
+  "trailingSlash": false
```

Comment:
In my testing, just setting `"cleanUrls": true` was enough to get the 404 page working, so I think we can narrow this down to that:

```suggestion
```

This is also less of a serious side effect than the `trailingSlash` option, which is more likely to conflict with desired behaviour. It’s still _definitely_ a bug on Vercel’s end that this is necessary, so curious to try to keep tabs on this so we know when this is fixed.